docs/articles/documentation/test-api/actions/drag-element.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title: Drag Element
+permalink: /documentation/test-api/actions/drag-element.html
+---
+# Drag Element
+
+There are two ways to drag a page element.
+
+* [Drag an Element by an Offset](#drag-an-element-by-an-offset)
+* [Drag an Element onto Another One](#drag-an-element-onto-another-one)
+
+## Drag an Element by an Offset
+
+```text
+drag( selector, dragOffsetX, dragOffsetY [, options] )
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`selector`             | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the webpage element being dragged. See [Selecting Target Elements](index.md#selecting-target-elements).
+`dragOffsetX`          | Numeric                                           | An X-offset of the drop coordinates from the mouse pointer's initial position.  
+`dragOffsetY`          | Numeric                                           | An Y-offset of the drop coordinates from the mouse pointer's initial position.
+`options` *(optional)* | Object                                            | A set of options that provide additional parameters for the action. See [Mouse Action Options](action-options.md#mouse-action-options).
+
+The following example demonstrates how to use the `drag` action with a slider.
+
+```js
+import { expect } from 'chai';
+import { Selector } from 'testcafe';
+
+const getSlider = Selector(id => document.querySelector('#developer-rating'));
+
+test('Drag slider', async t => {
+    await t.click('#i-tried-testcafe');
+
+    const slider = await getSlider();
+
+    expect(slider.value).to.equal(1);
+
+    await t.drag('.ui-slider-handle', 360, 0, { offsetX: 10, offsetY: 10 });
+
+    slider = await getSlider();
+
+    expect(slider.value).to.equal(7);
+});
+```
+
+Note that the `drag` action will not invoke integrated browser actions such as copying and pasting text.
```

Comment:
> We need to mention t.selectText.

So, we should say that despite dragging invokes selection, use t.selectText instead? @inikulin